### ISSUE TITLE
Support for Android 4.0+ and Landscape mode

### ISSUE THREAD

**Issue by venkat20390** [Nightonke/BlurLockView#4]
Dear Developer,
This library looks great. But, it will be more awesome if you could provide support from Android 4.0+ and landscape mode.

Regards,
Venkat.

**Comment by Nightonke** [comment]
Yep, I will consider this. Now the blur-calculate-method is not support android 4.0, but I will change it to another when I’m free. Thanks for suggestion.
